packages/rich-text-editor/src/RichTextEditor/RichTextEditor.scss

```diff
@@ -56,6 +56,14 @@
   border: $border-solid-border-width $border-solid-border-style $color-gray-500;
   background: $color-white;
   border-radius: $border-solid-border-radius;
+
+  div + .editor :global(.ProseMirror) {
```

Comment:
We generally try to avoid targetting element types like this, because if we for some reason decide to change `div` to a `span`, or something like that, this breaks silently.

What I'd do instead is create a class based on whether there are any controls, and as a result if there will be a toolbar.

Reply:
Changed it. Let me know what you think